scripts/quarantine_integration.yaml

```diff
@@ -1183,3 +1183,111 @@
   comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - zigbee.zboss_api.callback_api
+    - zigbee.zboss_api.alarm_api
+    - zigbee.osif.nvram
+    - zigbee.osif.crypto
+    - zigbee.osif.timer.ktimer
+    - zigbee.osif.serial.logger
+  platforms:
+    - nrf52833dk/nrf52833
+    - nrf52840dk/nrf52840
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - zigbee.osif.timer.counter
+    - zigbee.osif.serial.async
+  platforms:
+    - nrf52833dk/nrf52833
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - zigbee.osif.serial.basic
+  platforms:
+    - nrf52833dk/nrf52833
+    - nrf52840dk/nrf52840
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.template
+    - sample.zigbee.light_bulb.with_shell
+  platforms:
+    - nrf21540dk/nrf52840
+    - nrf52833dk/nrf52833
+    - nrf52840dk/nrf52840
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.network_coordinator
+  platforms:
+    - nrf21540dk/nrf52840
+    - nrf52840dk/nrf52840
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.light_bulb
+  platforms:
+    - nrf52833dk/nrf52833
+    - nrf52840dk/nrf52840
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.ncp
+    - sample.zigbee.shell
+  platforms:
+    - nrf21540dk/nrf52840
+    - nrf52833dk/nrf52833
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.ncp.usb.nrf21540dk_nrf52840
+    - sample.zigbee.shell.usb.nrf21540dk_nrf52840
+  platforms:
+    - nrf21540dk/nrf52840
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.ncp.usb.nrf52840dk_nrf52840
+    - sample.zigbee.shell.usb.nrf52840dk_nrf52840
+  platforms:
+    - nrf52840dk/nrf52840
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.shell
+  platforms:
+    - nrf5340dk/nrf5340/cpuapp
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.light_switch.fota
+    - sample.zigbee.light_switch.with_shell
+  platforms:
+    - nrf21540dk/nrf52840
+  comment: "Configurations excluded to limit resources usage in integration builds"
+
+- scenarios:
+    - sample.zigbee.light_switch.with_shell
+    - sample.zigbee.light_switch
+    - sample.zigbee.light_switch.current_measurement
+    - sample.zigbee.light_switch.with_shell
```

Comment:
This line is repeated. And what I understood from @mariusz-nordicsemi is that we want to skip it for all platforms, so what about moving it above after line 1217?

Reply:
Thanks, removed repeated line and moved at 1217.